rpm/SPECS/iotamanager.spec

```diff
@@ -161,3 +161,27 @@ fi
 %{_install_dir}
 
 %changelog
+
+* Fri Sep 09 2016 Daniel Moran <daniel.moranjimenez@telefonica.com> 1.0F.0
```

Comment:
Webstorm and OSX have a strange relationship; sometimes I'm trying to write in other window and it changes things without me realizing it. I'll fix it.